src/NuGet.Core/NuGet.Configuration/Utility/ConfigurationContants.cs

```diff
@@ -11,6 +11,8 @@ public static class ConfigurationConstants
 
         public static readonly string AllowUntrustedRoot = "allowUntrustedRoot";
 
+        public static readonly string AllowInsecureConnections = "AllowInsecureConnections";
```

Comment:
Should this be camelcased like "allowUntrustedRoot"?

Reply:
Good catch! Fixed. Thanks!